app/javascript/src/stylesheet/application.scss

```
@@ -123,3 +123,12 @@ div.sakes-form-label + div input:checked[type="checkbox"] + label::after {
   @extend .col-lg-9;
   @extend .col-12;
 }
+
+//------ header ------
+// ロゴ
+.sakazuki-logo {
```

名前からわかるコメントはいらない
//--header--のあとは他に合わせて空行ほしい
他に合わせてheader prefixつけるべき

```suggestion
//------ header ------

.sakazuki-header-logo {
```